PWGCF/EbyEFluctuations/Tasks/EbyEmeanPtFluc.cxx

```diff
@@ -0,0 +1,777 @@
+
+    hist.add("QA/after/Kaon/h_Mult_ka", "Multiplicity Kaon", kTH1D, {axisMult});
+    hist.add("QA/after/Kaon/h_Pt_ka", "p_{T} (Kaons) TPC and TPC+TOF", kTH1D, {axisPt});
+    hist.add("QA/after/Kaon/h_rap_ka", "y (Kaons) TPC and TPC+TOF", kTH1D, {axisY});
+    hist.add("QA/after/Kaon/h2_Pt_rap_ka", "p_{T} vs y (Kaons)", kTH2D, {{axisY}, {axisPt}});
+    hist.add("QA/after/Kaon/h2_DcaZ_ka", "DCA_{z} Kaons", kTH2D, {{axisPt}, {axisDCAz}});
+    hist.add("QA/after/Kaon/h2_DcaXY_ka", "DCA_{xy} Kaons", kTH2D, {{axisPt}, {axisDCAxy}});
+    hist.add("QA/after/Kaon/h2_TPCNsigma_ka", "n #sigma_{TPC} Kaons",
+             kTH2D, {{axisP}, {axisTPCNsigma}});
+    hist.add("QA/after/Kaon/h2_TOFNsigma_ka", "n #sigma_{TOF} Kaons",
+             kTH2D, {{axisP}, {axisTOFNsigma}});
+    hist.add("QA/after/Kaon/h2_TpcTofNsigma_ka", "n #sigma_{TPC} vs n #sigma_{TOF} Kaons",
+             kTH2D, {{axisTPCNsigma}, {axisTOFNsigma}});
+    hist.add("QA/after/Kaon/h2_TPCSignal_ka_a", "TPC Signal Kaons (after)",
+             kTH2D, {{axisP}, {axisTPCSignal}});
+    hist.add("QA/after/Kaon/h2_TOFSignal_ka_a", "TOF Signal Kaons (after)",
+             kTH2D, {{axisP}, {axisTOFSignal}});
+    hist.add("QA/after/Kaon/h2_ExpTPCSignal_ka_a", "Expected TPC Signal Kaons (after)",
+             kTH2D, {{axisP}, {axisTPCSignal}});
+    hist.add("QA/after/Kaon/h2_ExpTOFSignal_ka_a", "Expected TOF Signal Kaons (after)",
+             kTH2D, {{axisP}, {axisTOFSignal}});
+
+    hist.add("QA/after/Proton/h_Mult_pr", "Multiplicity Proton", kTH1D, {axisMult});
+    hist.add("QA/after/Proton/h_Pt_pr", "p_{T} (Protons) TPC and TPC+TOF", kTH1D, {axisPt});
+    hist.add("QA/after/Proton/h_rap_pr", "y(Protons) TPC and TPC+TOF", kTH1D, {axisY});
+    hist.add("QA/after/Proton/h2_Pt_rap_pr", "p_{T} vs y (Protons)", kTH2D, {{axisY}, {axisPt}});
+    hist.add("QA/after/Proton/h2_DcaZ_pr", "DCA_{z} (Protons)", kTH2D, {{axisPt}, {axisDCAz}});
+    hist.add("QA/after/Proton/h2_DcaXY_pr", "DCA_{xy} (Protons)", kTH2D, {{axisPt}, {axisDCAxy}});
+    hist.add("QA/after/Proton/h2_TPCNsigma_pr", "n #sigma_{TPC} (Protons)",
+             kTH2D, {{axisP}, {axisTPCNsigma}});
+    hist.add("QA/after/Proton/h2_TOFNsigma_pr", "n #sigma_{TOF} (Protons)",
+             kTH2D, {{axisP}, {axisTOFNsigma}});
+    hist.add("QA/after/Proton/h2_TpcTofNsigma_pr", "n #sigma_{TPC} vs n #sigma_{TOF} (Protons)",
+             kTH2D, {{{axisTPCNsigma}, {axisTOFNsigma}}});
+    hist.add("QA/after/Proton/h2_TPCSignal_pr_a", "TPC Signal Protons (after)",
+             kTH2D, {{axisP}, {axisTPCSignal}});
+    hist.add("QA/after/Proton/h2_TOFSignal_pr_a", "TOF Signal Protons (after)",
+             kTH2D, {{axisP}, {axisTOFSignal}});
+    hist.add("QA/after/Proton/h2_ExpTPCSignal_pr_a", "Expected TPC Signal Protons (after)",
+             kTH2D, {{axisP}, {axisTPCSignal}});
+    hist.add("QA/after/Proton/h2_ExpTOFSignal_pr_a", "Expected TOF Signal Protons (after)",
+             kTH2D, {{axisP}, {axisTOFSignal}});
```

Comment:
Consider checking the documentation
You should able to replicate the full content of a `HistogramRegistry` directory without the need of adding histograms one by one
That would allow you to not unnecessarily replicate code
Check if it could apply to your case